protected/humhub/modules/content/components/ContentContainerModule.php

```diff
@@ -219,4 +219,15 @@ protected function getGlobalPermissions()
         return [];
     }
 
+    /**
+     * This method is called to determine classes of Content models which can be posted on wall.
+     *
+     * @param ContentContainerActiveRecord|null $contentContainer
+     * @return array
+     */
+    public function getContentClasses(?ContentContainerActiveRecord $contentContainer = null): array
```

Comment:
I am a bit unsure if we should pass the `ContentContainer` as a parameter here.

Would each module then have to check itself whether it is activated in the respective ContentContainer?

For (see next comment) the CC would be handy of course.

Reply:
@luke- I thought if a model should to be implemented only for Space and not for User.
If you think we don't need this, so all models will be implemented and for Space and for User then we can remove the param, let me know to remove.